packages/odyssey-react-mui/src/Pagination/Pagination.tsx

```diff
@@ -286,7 +286,7 @@ const Pagination = ({
   );
 
   return variant === "paged" ? (
-    <PaginationContainer>
+    <PaginationContainer data-testId={`${variant}-pagination`}>
```

Comment:
We shouldn't be using test IDs anywhere in our tests. If we can't select with accessibility selectors from Testing Library such as `byRole`, then users cannot see those elements either.